lesson_13/xml_task.py

```diff
@@ -0,0 +1,41 @@
+
+import xml.etree.ElementTree as ET
+import logging
+
+
+"""Завдання 3:
+Для файла ideas_for_test/work_with_xml/groups.xml створіть функцію пошуку по group/number і повернення 
+значення timingExbytes/incoming результат виведіть у консоль через логер на рівні інфо
+"""
+
+xml_file = '/lesson_13/numbers.xml'
+#Variable with path to xml
+
+def find_value_in_tree(xml_file):
```

Comment:
method should accept defined group and according defined group we should search for target value